charts/oes/templates/rbac/oes-init-rolebinding.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if (eq .Values.installationMode "OES-AP") -}}
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
+kind: RoleBinding # ClusterRole if you have cluster access
```

Comment:
If this can be ClusterRoleBinding, we should make this configurable as a user should not edit templates